app/assets/javascripts/lifeitup_layout.js

```diff
@@ -61,7 +61,10 @@ function layout_resizer () {
 // init popovers
 
 $(function () {
-  $('[data-toggle="popover"]').popover()
+  $('body').popover({
+    selector: '[data-toggle="popover"]',
+    trigger: 'focus'

```

Comment:
I wouldnt "get rid of it".. since like for example, since I gave you the files at the beginning I made more enhancements in Lifeitup and a few days ago I merged the new changes from Lifeitup to which made everything look the way it looks now... and eventually there might me more improvement coming from Lifeitup.
It saves me time, since im more active in Lifeitup than here, because I work on that in my private time, and at Portus from SUSE I have very little official time to spend: Manager and SCC are my priorities. If I didnt had Lifeitup portus would only look like any other bootstrap page.

So, from my point of view we need to keep it, move it to Vendor and keep working from that on top.